requirements.txt

```diff
@@ -1,4 +1,5 @@
 pandas>=2.1.2
 numpy>=1.26.1
-seaborn>=0.13.1
+seaborn>=0.13.0
 matplotlib>=3.8.1
+scikit-learn
```

Comment:
[Sugestion] It could be good to add `scikit-learn` minimum version?

Reply:
Done